static/js/components/admin/EditChannelContributorsForm.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import React from "react"
+import { Link } from "react-router-dom"
+
+import Card from "../Card"
+import MembersNavbar from "./MembersNavbar"
+
+import { profileURL } from "../../lib/url"
+
+import type { ChannelContributorsForm } from "../../flow/discussionTypes"
+
+export default class EditChannelContributorsForm extends React.Component<
+  *,
+  void
+> {
+  props: {
+    form: ChannelContributorsForm
+  }
+
+  render() {
+    const { form } = this.props
+    return (
+      <form className="form contributors-form">
```

Comment:
This follows from the Slack convo, but I'm not sure that all of this should be part of a form or that this data should be part of a form object. I think we should just use forms for form-like behavior. The only user inputs that I imagine will be part of this eventually are the inputs for a new contributor/moderator. If we just have delete buttons that will sit on each row for each user, those can just be buttons with onClick events. Is there some more involved user input that I'm not aware of?

Reply:
Are you saying I should remove the `<form />` or that we should merge the `...Form` into the `...Page` object?

Reply:
I would say remove the form and pass the contributor/moderator data into each component as standalone arrays